model/host/hostutil.go

```diff
@@ -453,6 +451,9 @@ func (h *Host) ProvisioningUserData(settings *evergreen.Settings, creds *certdep
 	if h.Distro.IsWindows() {
 		var setupScriptCmds []string
 		if h.Distro.BootstrapSettings.FetchProvisioningScript {
+			// If we're fetching the provisioning script, we can just execute
+			// the setup script commands directly rather than write it to an
```

Comment:
> rather than write it to an intermediate file.

I mean, we are _eventually_ writing to an intermediate file.

Reply:
I clarified the wording a bit.